endorser/api/db/migrations/versions/rename_author_did_to_creddef_author_did__b1e41d72f00e.py

```diff
@@ -0,0 +1,35 @@
+"""rename author_did to creddef_author_did and issuer_did to schema_issuer_did
+
+Revision ID: b1e41d72f00e
+Revises: 9a8ef028a751
+Create Date: 2024-09-12 00:02:34.371864
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel
+
+
+# revision identifiers, used by Alembic.
+revision = 'b1e41d72f00e'
+down_revision = '9a8ef028a751'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('allowedcredentialdefinition', sa.Column('schema_issuer_did', sqlmodel.sql.sqltypes.AutoString(), nullable=False))
+    op.add_column('allowedcredentialdefinition', sa.Column('creddef_author_did', sqlmodel.sql.sqltypes.AutoString(), nullable=False))
+    op.drop_column('allowedcredentialdefinition', 'issuer_did')
+    op.drop_column('allowedcredentialdefinition', 'author_did')
+    # ### end Alembic commands ###
```

Comment:
This migration appears to be **destructive**, adding the columns with the new name and dropping the columns with the old name, and on the surface it looks like the existing data in the existing columns would be deleted along with the columns.

Reply:
This should be using `alter_column` so the columns are simply renamed and the data and all other attributes are retained.

Reply:
It would be interesting to understand why the auto-generation chose to drop and add rather than rename the existing column.

Reply:
Good catch, I'll look into the documentation. Worst case I can do it manually.